string object_id
string object_type
geometry_msgs/Pose aruco_pose
---
int8 flag